scala/spark-wordcount/README.md

```diff
@@ -38,10 +52,39 @@ Create a Cloud DataProc instance:
 
 ## Build the jar
 
-The Spark job is assembled into a fat jar with all of its dependencies. To build, run:
+The Spark application is assembled into a fat jar with all of its dependencies. To build, run:
 
     mvn assembly:assembly
 
+## Test your job locally with Cloud Bigtable Emulator (optional but recommended)
```

Comment:
`Test your job locally with Bigtable emulator (optional but recommended)`

Reply:
Also rename the existing `Test your job locally (optional but recommend)` title to 
`Test your job locally using a real Cloud Bigtable instance (optional but recommend)`

Reply:
think it got fixed with the other changes